src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,16 @@
 import React from 'react';
+import { User } from '../../types/users';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
+type Props = {
+  user: User | null
```

Comment:
you handle it in upper component
```suggestion
  user: User;
```